openSpotServer/parkingSpot/DEMO_ParkingSpotMask.py

```diff
@@ -0,0 +1,15 @@
+import cv2
+from parkingSpots import *
```

Comment:
I prefer to do named imports so
 `from parkingSpots import (spot_16, spot_17, spot_18)`

It promotes readability in the code since you can easily reference where a variable comes from.

We can also just do
```
import parkingSpots
...
image = get_mask_of_parking_spot(image, parkingSpots.spot_16, True)
image = get_mask_of_parking_spot(image, parkingSpots.spot_17, True)
...
```